lib/Makefile

```diff
@@ -49,6 +49,7 @@ ifeq ($HAVE_COLORNEVER, 1)
 GREP_OPTIONS += --color=never
 endif
 GREP = grep $(GREP_OPTIONS)
+SED_ERE_OPT ?= -E
```

Comment:
I'm wondering if it would be better to centralize these changes and options into a second variable `SED`, such as for example :
`SED ?= sed $(SED_ERE_OPT)`

Reply:
Some of the sed commands use -E while others do not

Reply:
We could switch to `-E` for the other commands and simplify the logic, but let's do that in a separate PR.

I'd rather not add functional changes to a PR that is currently a functional no-op when the user doesn't override `SED_ERE_OPT`.